extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -175,6 +183,9 @@ class AmpBridPlayer extends AMP.BaseElement {
 
     installVideoManagerForDoc(element);
     Services.videoManagerForDoc(element).register(this);
+
+    /** Register Brid actions */
+    this.registerDefaultAction(() => this.destroy(), BridActions.DESTROY);
```

Comment:
This should probably use `registerAction`. `registerDefaultAction` would call `destroy` when using only the element's id without an action name like `<button on="tap:my-brid-player">`.

```suggestion
    this.registerAction(BridActions.DESTROY, () => this.destroy());
```